src/deluge/gui/views/session_view.cpp

```diff
@@ -1827,10 +1827,6 @@ void SessionView::renderOLED(uint8_t image[][OLED_MAIN_WIDTH_PIXELS]) {
 
 void SessionView::redrawNumericDisplay() {
 	UI* currentUI = getCurrentUI();
```

Comment:
I think this becomes stale now and no longer required unless you want to use currentUI down below in the function in place of the other getCurrentUI() calls (I would do that).

Reply:
Fixed, thanks!